xsd/netex_part_1/part1_ifopt/netex_ifopt_equipmentAccess_version.xsd

```diff
@@ -1426,6 +1511,31 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether equipment is suitable for cycles.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
+			<xsd:element name="MarkingStatus" type="MarkingStatusEnumeration" default="good" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Status of the ground marking of the crossing</xsd:documentation>
+				</xsd:annotation>
+			</xsd:element>
+			<xsd:element name="VibratingCrossingAids" type="xsd:boolean" minOccurs="0">
+				<xsd:annotation>
+					<xsd:documentation>Indicates the presence of a device delivering vibrations to facilitate the crossing of visually impaired people. May be associated with the OSM traffic_signals:floor_vibration and traffic_signals:vibration</xsd:documentation>
```

Comment:
Still OSM references.

Reply:
you want to generally avoid it? Or have it as an example (my preference)